beacon-chain/p2p/encoder/ssz.go

```diff
@@ -43,21 +43,6 @@ func (e SszNetworkEncoder) doEncode(msg interface{}) ([]byte, error) {
 	return ssz.Marshal(msg)
 }
 
-// Encode the proto message to the io.Writer.
-func (e SszNetworkEncoder) Encode(w io.Writer, msg interface{}) (int, error) {
```

Comment:
No longer relevant. We only used it in tests. Updated tests to use `EncodeGossip`